### PR TITLE
Benja make tbepanels 978

### PR DESCRIPTION
**Pull request by Shriukan33**
This PR introduces TopicBlogPanels

Panels are short html snippets that can be embedded in other topicblog contents through our markdown (using `[[panel:]]((panel-slug))`) or using the templatetag synthax `{% load panels %}{% panel 'panel-slug' %} `

The panels work the same way all other topicblog items work with a publication / versioning system.

The provided template is email client-friendly, as well as mobile friendly (it removes the image on smaller screens)

### CODE REVIEW

**Comment by JeffAbrahamson** [comment]
Modifies tn_links.py without modifying test_tn_links.py.

Leaves a comment about a bug (that it apparently fixes), merely adding a comment about the comment.

**Comment by JeffAbrahamson** [comment]
Three of the seven commits have no issue tag.

**Comment by Shriukan33** [comment]
Updated tests and commit messages

File: transport_nantes/topicblog/test_tn_links.py
Comment (JeffAbrahamson) [inline transport_nantes/topicblog/test_tn_links.py:58]: This isn't related in any way to the task at hand, even though it's a good fix.  It should really be a separate commit (with or without issue... ;-) and with commit message about test the tn-md news tag.

Meanwhile, aside, you have added the "panel" directive but haven't tested it.

File: transport_nantes/topicblog/urls.py
Comment (JeffAbrahamson) [inline transport_nantes/topicblog/urls.py:14]: This is a good commit, but why is it in this PR?  This PR is long and has some minor issues, whereas this commit would have just been a simple approve (and so not needed to be re-read on next iteration).

**Review by JeffAbrahamson — CHANGES_REQUESTED** [review]

File: transport_nantes/topicblog/urls.py
Comment (Shriukan33) [reply transport_nantes/topicblog/urls.py:14]: This is because I've used the new url list format in this PR, creating a "topicblogpanel_urls" list in the next commit. So we would have to merge this change before TBPanels.

I can open a separate PR, but I thought this was a well isolated commit, enough to be reviewed once and approved without having to look at it twice. :)

File: transport_nantes/topicblog/test_tn_links.py
Comment (Shriukan33) [reply transport_nantes/topicblog/test_tn_links.py:58]: I did actually test it, see : 
https://github.com/transport-nantes/tn_web/blob/6c5fa78caa585ce96b7b3b82e1a7073ab969b29f/transport_nantes/topicblog/templatetags/test_templatags.py#L202-L215

I didn't put it in the test_tn_links because to the difference of the other md stuff, the rendering aspect does rely on a database object witch is created in setUp. It was out of place in test_tn_links.py, in my opinion.

Comment (Shriukan33) [reply transport_nantes/topicblog/test_tn_links.py:58]: Also, made issue + separate commit for the news tag, as requested :)

File: transport_nantes/topicblog/urls.py
Comment (JeffAbrahamson) [reply transport_nantes/topicblog/urls.py:14]: It would have been approved already if separate, and you would have quietly rebased this one on it so as to use it...

File: transport_nantes/topicblog/test_tn_links.py
Comment (JeffAbrahamson) [reply transport_nantes/topicblog/test_tn_links.py:58]: Testing the markdown and testing the templatetag are two different things.  The markdown must be tested.

Comment (Shriukan33) [reply transport_nantes/topicblog/test_tn_links.py:58]: > {% tn_markdown "[[panel:]]((test-panel))" %} 

tn_markdown is the tag we use to make text go through our parser, it's **not** related to tbpanels, which have their own template tag, namely {% panel 'slug-here' %}

**Comment by Shriukan33** [comment]
Added comments about where the test is located

**Comment by Shriukan33** [comment]
Requesting re-review

**Review by JeffAbrahamson — APPROVED** [review]
LGTM.

**Pull request merged**
